src/app/pages/forms-and-validation/fix-length-passcode/services/customvalidation.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { CustomvalidationService } from './customvalidation.service';
+
+describe('CustomvalidationService', () => {
+    let service: CustomvalidationService;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({});
+        service = TestBed.inject(CustomvalidationService);
+    });
+
+    it('should be created', () => {
+        expect(service).toBeTruthy();
+    });
```

Comment:
I feel like we should be able to do more than 'renders successfully' tests for these form examples

Reply:
Yeah these could be be enhanced to include something along the lines of 'it should grayed out the description when password meets criteria'